pages/index.tsx

```diff
@@ -144,69 +147,95 @@ export default function Home() {
-                </div>
-              </div>
-            </a>
-            <a href="https://www.notion.so/s-h-l/Walkthrough-058a7ba0a8fe4d798370e4f6a5fda8b0#b9f3a9b9f2c645fb82e9633a2e44ca19">
-              <div className={"game-tile " + (gameTileCompletionStates[2] ? "completed" : "")}>
-                <div className="tooltip">
-                  Use the faucet to get .1 $MATIC
-                </div>
-              </div>
-            </a>
-            <a href="https://www.notion.so/s-h-l/Walkthrough-058a7ba0a8fe4d798370e4f6a5fda8b0#dfbd8c7587504d72b93cfa9b1ed3d822">
-              <div className={"game-tile " + (gameTileCompletionStates[3] ? "completed" : "")}>
-                <div className="tooltip">
-                  Send 100 $FWEB3 tokens to someone
-                </div>
-              </div>
-            </a>
-            <a href="https://www.notion.so/s-h-l/Walkthrough-058a7ba0a8fe4d798370e4f6a5fda8b0#1683ea6108ae41e49a6cfa3a30fdc0a7">
-              <div className={"game-tile " + (gameTileCompletionStates[4] ? "completed" : "")}>
-                <div className="tooltip">
-                  Mint a Fweb3 NFT
-                </div>
-              </div>
-            </a>
-            <a href="https://www.notion.so/s-h-l/Walkthrough-058a7ba0a8fe4d798370e4f6a5fda8b0#50bdb9a3ed904aa58b0e71979e7f52d6">
-              <div className={"game-tile " + (gameTileCompletionStates[5] ? "completed" : "")}>
-                <div className="tooltip">
-                  Burn at least one $FWEB3 token
-                </div>
-              </div>
-            </a>
-            <a href="https://www.notion.so/s-h-l/Walkthrough-058a7ba0a8fe4d798370e4f6a5fda8b0#de8d1ab1d12f408b96b9f4d7156b3959">
-              <div className={"game-tile " + (gameTileCompletionStates[6] ? "completed" : "")}>
-                <div className="tooltip">
-                  Swap a $FWEB3 token for some $MATIC
-                </div>
-              </div>
-            </a>
-            <a href="https://www.notion.so/s-h-l/Walkthrough-058a7ba0a8fe4d798370e4f6a5fda8b0#f9d992dc1327486eba0e417f53ac7753">
-              <div className={"game-tile " + (gameTileCompletionStates[7] ? "completed" : "")}>
-                <div className="tooltip">
-                  Vote on a Fweb3 poll
-                </div>
-              </div>
-            </a>
-            <a href="https://www.notion.so/s-h-l/Walkthrough-058a7ba0a8fe4d798370e4f6a5fda8b0#034d12e44d9e4ee693a193dc4b1e7ab0">
-              <div className={"game-tile " + (gameTileCompletionStates[8] ? "completed" : "")}>
-                <div className="tooltip">
-                  Write and deploy a smart contract
-                </div>
-              </div>
-            </a>
+            <div
+              onClick={() => setActiveDot(0)}
+              className={
+                "game-tile js-dot0 " +
+                (gameTileCompletionStates[0] ? "completed" : "")
+              }
+            >
```

Comment:
```suggestion
            <div
              data-test-id="step0"
              onClick={() => setActiveDot(0)}
              className={
                "game-tile" +
                (gameTileCompletionStates[0] ? "completed" : "")
              }
            >
```

for testing purposes, you'll probably want to use custom [data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) to target UI elements instead of classes like `js-dot0`

https://docs.cypress.io/guides/references/best-practices#Selecting-Elements